src/paths.cpp

```diff
@@ -66,6 +66,7 @@ wxString GetConfigFile()
 //     wxStandardPaths::GetResourcesDir
 //     wxStandardPaths::GetAppDir
 // Fortunately these should always return the exe directory.
+// TODO: Investigate using wxStandardPaths::DontIgnoreAppSubDir with wxWidgets 2.9.1+
 #if defined(__WXDEBUG__) && defined(__WXMSW__)
```

Comment:
Would it be okay to defer that for a follow-up pull request? I'm afraid this one is getting a bit big, and I've had to stack more changes on top of it to avoid conflicts - naturally a lot of things depend on a working build as a baseline :)

The change you describe seems simple but I'd want to make sure it was working on all platforms before sending it out, and it feels a little out of scope of compilation fixes.

If not, LMK and I'll look into it here.